website/content/en/docs/scorecard/custom-tests.md

```diff
@@ -0,0 +1,185 @@
+# Custom Tests using Operator SDK Scorecard
+
+This guide outlines the steps which can be followed to extend the existing scorecard tests and implement operator specific custom tests. 
+
+## Requirements:
+
+Following are requirements for the operator project which would be checked by the scorecard.
+
+1. [Docker][docker_tool] version 17.03+
+2. Access to a Kubernetes v1.11.3+ cluster
+
+**Note**: It is recommended that you have your project upgraded to the latest SDK release version before implementing the following steps to extend Scorecard's basic tests.
+
```

Comment:
IMO the requirements could be removed since they are the pre-requirements to work with SDK